### PR TITLE
Add an health endpoint to OG

### PR DESCRIPTION
**Pull request by otherview**
### Why this change is needed

https://github.com/obscuronet/obscuro-internal/issues/2018

### What changes were made as part of this PR

Please provide a high level list of the changes made

### PR checks pre-merging

Please indicate below by ticking the checkbox that you have read and performed the required
[PR checks](https://github.com/obscuronet/obscuro-internal/blob/main/dev-ops-docs/dev-pr-checks.md)

- [ ] PR checks reviewed and performed

### CODE REVIEW

File: integration/obscurogateway/obscurogateway_test.go [hunk 1]
```diff
@@ -67,7 +71,11 @@ func TestObscuroGateway(t *testing.T) {
 
 	// make sure the server is ready to receive requests
 	// TODO Implement health endpoint
```
Comment (zkokelj) [inline integration/obscurogateway/obscurogateway_test.go:74]: Remove this todo

**Review by zkokelj — APPROVED** [review]
LGTM

File: integration/obscurogateway/obscurogateway_test.go (hunk 1 above)
Comment (otherview) [reply integration/obscurogateway/obscurogateway_test.go:74]:
```suggestion
```

**Review by tudor-malene — APPROVED** [review]
lgtm

**Pull request merged**